### ISSUE TITLE
Unexpected GraphQL Error Int cannot represent non 32-bit signed integer value

### ISSUE THREAD

**Issue by volker48** [dagster-io/dagster#4796]
Dagster version 0.12.5.

I'm receiving the following unexpected GraphQL error when trying to view the status of a run.

  ```
Operation name: RunLogsQuery

Message: Int cannot represent non 32-bit signed integer value: 2338768730

Path: 

Locations: 

Stack Trace:
  File "/usr/local/lib/python3.7/site-packages/graphql/execution/executor.py", line 481, in complete_value_catching_error
    exe_context, return_type, field_asts, info, path, result
  File "/usr/local/lib/python3.7/site-packages/graphql/execution/executor.py", line 563, in complete_value
    return complete_leaf_value(return_type, path, result)
  File "/usr/local/lib/python3.7/site-packages/graphql/execution/executor.py", line 626, in complete_leaf_value
    serialized_result = return_type.serialize(result)
  File "/usr/local/lib/python3.7/site-packages/graphql/type/scalars.py", line 32, in coerce_int
    ("Int cannot represent non 32-bit signed integer value: {}").format(value)
```

The run had a duration of 1:16:36 and did finish successfully despite this error I was just unable to view the run in the UI.

**Comment by sryza** [comment]
@hellendag @prha any idea what's going on here?

**Comment by prha** [comment]
Yeah, I think we're using a cursor with graphql type Int, which might be insufficient for runs with very long logs.  I think we can resolve by changing the type and coercing to the correct offset in the python resolver.

**Comment by prha** [comment]
(this is a guess, it's hard to tell which field is triggering the error)

**Comment by prha** [comment]
@volker48 what version of graphql / graphene do you have installed?

**Comment by volker48** [comment]
@prha I'm using poetry and in my dependencies I had `dagster-graphql = "^0.12.4"`, which ended up getting me graphene is 2.1.9 and graphql-core 2.3.2 in my poetry.lock file.

**Comment by prha** [comment]
Hmm, that's what I have also, but I'm not triggering the exact same error.  I wonder if this is a different field than the cursor field, which is what I was originally thinking.  This is only happening for these very long runs, correct?  Any chance that you'd be able to share a zip of the debug run payload (might be able to download via the 'Debug File' button in the top right).

**Comment by volker48** [comment]
Hmm so I just loaded up one of the old runs that was doing this, but we've since upgraded to dagster 0.12.10 and the run loaded fine. It could also just be because the run is finished now and not in progress I'm not sure. I was able to grab the debug file, but I'll need to get approval to share it.

**Comment by alangenfeld** [comment]
https://github.com/dagster-io/dagster/pull/4550 fixed in `0.12.7`
